index.html

```diff
@@ -0,0 +1,180 @@
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="home">
+      <div class="home-text">
+        <h2>"The IT World Needs You"</h2>
+        <h1>We Build, We Develop</h1>
+        <p class="home-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem enim obcaecati praesentium veniam nisi suscipit error, modi quisquam eaque fugit asperiores at itaque amet blanditiis mollitia pariatur?</p>
+        <div class="date-time">
+          <h3>2023.07.05(WED) ~ 06(THU)</h3>
+          <p>@ iHub, Senteu Plaza Nairobi</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="main-programs">
+      <h2>Our Programs</h2>
+      <div class="program-cards">
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/react-2.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>React</h4>
+          </div>
+          <div class="logo-description">
+            <p>React is a free JavaScript library for building user interfaces based on components.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/rails-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Ruby on Rails</h4>
+          </div>
+          <div class="logo-description">
+            <p>Rails is a framework that provides default structures for databases.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/devops-2 (1).svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Devops</h4>
+          </div>
+          <div class="logo-description">
+            <p>DevOps improves and shortens the systems development life cycle.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/kotlin-1.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Kotlin</h4>
+          </div>
+          <div class="logo-description">
+            <p>Kotlin is a cross-platform high-level programming language with type inference.</p>
+          </div>
+        </div>
+        <div class="program-card">
+          <div class="logo">
+            <img src="./images/vue-9.svg" alt="react logo">
+          </div>
+          <div class="logo-title">
+            <h4>Vue.js</h4>
+          </div>
+          <div class="logo-description">
+            <p>Vue.js is a JavaScript framework for building user interfaces and single-page applications.</p>
+          </div>
+        </div>
+      </div>
+      <a href="#">Join Our Program</a>
+    </section>
+
+    <section class="speakers">
+      <h2>Featured Speakers</h2>
+      <div id="speaker-cards"></div>
+    </section>
```

Comment:
- [ ] Good job so far your contributor section is loading dynamically :clap:  But the design is a bit different compared to the original design. :call_me_hand:   Please make sure that your contributor section has use the same font, `height`, `width `, and spacing `padding/margin `. And  your `background color`  is white, please change it  to the color defined on the original design  in order to make your  design similar to the original design :+1: 
 Check the screenshots below :point_down:  to see the differences.

#### Orignal design
![Screenshot from 2022-10-28 18-19-46](https://user-images.githubusercontent.com/6615394/198707921-f81e8d82-3ba1-46d2-868b-a34d04408983.jpg)

#### Your design
![speakers](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/e8fae51c-727b-45cc-9678-0fab0631303c)

Reply:
Fixed